src/postgres/README.md

```diff
@@ -11,3 +11,8 @@ Load data into tables
 ```sh
 psql 'dbname=imdb user=imdb options=--search-path=imdb' -f load_tables.sql
 ```
+
+Update title_basics table to be able do search
+```sh
+psql 'dbname=imdb user=imdb options=--search-path=imdb' -f patch-0.1.sql
```

Comment:
uh oh. What is this? I wasn't using postgres part of this project. I was importing via step1 in https://github.com/tarasglek/py_gql_jaeger_workshop

Reply:
I'm guessing you just added this column to db imported in this step. Should probably delete the rest of the files in postgres/ dir

Reply:
Yes, we can delete the rest of the files in this directory